scripts/update/00-checkapt.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Checks if required functions are loaded before continuing with the update scripts.
+# Function should be loaded on the next run (by `box`) without further need for interaction.
+
+if ! command -v apt_install > /dev/null 2>&1 ; then
+    echo
+    echo "Due to internal restructuring please restart \`box update\`. You should only have to do this once."
+    kill -13 `ps --pid $$ -oppid=`
```

Comment:
Codacy found an issue: [Use $(...) notation instead of legacy backticked `...`.](https://app.codacy.com/manual/flying_sausages/swizzin/pullRequest?prid=6240189)